src/PageTypes/BasePage.php

```diff
@@ -26,6 +26,9 @@
  * Please note: you should generally always extend `Page` with your custom page types. Refrain from subclassing
  * `BasePage` directly. Doing so will omit `Page` from your custom class's hierarchy, and this can have unintended
  * side effects where modules rely on modifying things on the `Page` class, as well as with templates and themes.
+ * @method SilverStripe\ORM\ManyManyList<BasePage> RelatedPagesThrough()
```

Comment:
This would be a [`ManyManyThroughList`](https://api.silverstripe.org/5/SilverStripe/ORM/ManyManyThroughList.html)

Reply:
Updated